tests/regression/README.md

```diff
@@ -133,3 +134,7 @@ Configures and creates a new s2n-tls configuration with a specified security pol
 ### test_rsa_handshake
 
 Performs an RSA handshake in s2n-tls and validates the handshake process utilizing rsa_4096_sha512.
+
+### test_session_resumption
+
+Performs an RSA handshake, setting a session ticket in the client. Then, measures the resumption utilizing the session ticket in a second RSA handshake and asserts that the client has resumed the session.
```

Comment:
This is worded kind of weirdly. I would say something like "Performs a handshake with server authentication. Then performs a resumption handshake using the session ticket obtained from the previous handshake."